docs/fluvio/concepts/batching.mdx

```diff
@@ -6,10 +6,87 @@ title: "Batching"
 Fluvio producers try to send records in batches to reduce the number of messages sent and improve throughput. Each producer has some configurations that can be set to improve performance for a specific use case. For instance, they can be used to reduce disk usage, reduce latency, or improve throughput.
 As of today, batching behavior in Fluvio Producers can be modified with the following configurations:
 
-- `batch_size`: Indicates the maximum amount of bytes that can be accumulated in a batch.
-- `linger`: Time to wait before sending messages to the server. Defaults to 100 ms.
+- `max_request_size`: Indicates the maximum amount of bytes that the producer can send in a single request. If the record is larger than the max request size, the producer will fail to send the record. Defaults to 1048576 bytes.
```

Comment:
should we mention this is uncompressed?

Reply:
Sure, I added `Only the uncompressed size of the record is considered.` wdyt?